abis_mapping/types/spatial.py

```diff
@@ -8,6 +8,7 @@
 # Third-party
 import shapely
 import shapely.ops
+import numpy as np
```

Comment:
Add numpy to `[tool.poetry.dependencies]` in pyproject.toml since we are adding a direct import of it - 
`$ poetry add "numpy^1.0.0"`

Reply:
yeah i wasn't sure on this one since it is closely tied to shapely. Have added it.